autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers() abort
-    return ['go', 'golint', 'govet']
+    return ['go', 'golint', 'govet', 'gometalinter']
```

Comment:
> I don't think it should be preferred, it just runs those same tools.

But it means that checks are then run twice / duplicated, no?
It really sounds like `gometalinter` should be enabled alone by default, if it is installed?! (given that it also reports compile errors)